src/neptune/cli/status.py

```diff
@@ -20,21 +20,25 @@
 import textwrap
 from pathlib import Path
 from typing import (
-    Optional,
+    TYPE_CHECKING,
     Sequence,
 )
 
-from neptune.cli.abstract_backend_runner import AbstractBackendRunner
-from neptune.cli.container_manager import ContainersManager
-from neptune.cli.utils import (
-    get_offline_dirs,
-    get_qualified_name,
+from neptune.cli.collect import collect_containers
+from neptune.cli.containers import (
+    AsyncContainer,
+    OfflineContainer,
 )
+from neptune.cli.utils import get_qualified_name
 from neptune.constants import OFFLINE_NAME_PREFIX
 from neptune.envs import PROJECT_ENV_NAME
 from neptune.internal.backends.api_model import ApiExperiment
 from neptune.internal.utils.logger import logger
```

Comment:
Once https://github.com/neptune-ai/neptune-client/pull/1611 is merged we'll need to adjust this and similar code fragments, right?